rust/ql/test/query-tests/unusedentities/UnusedVariable.expected

```diff
@@ -1,10 +1,7 @@
 | main.rs:25:9:25:9 | a | Variable is not used. |
 | main.rs:90:13:90:13 | d | Variable is not used. |
-| main.rs:114:9:114:9 | k | Variable is not used. |
```

Comment:
The `SPURIOUS` comments should be removed in `main.rs`.

Reply:
@redsun82 let me know if you're already on this, I have a bit of follow-up to do after this PR anyway so I could fix this as well?

Reply:
Perhaps the comments should be converted to inline test expecations?

Reply:
Here's the fix: https://github.com/github/codeql/pull/17744

Reply:
And there's already an issue for converting them into inline expectations, but that's waiting on *having* inline expectations for query tests.  Is that work finished yet?

Reply:
No, not finished yet.